File: src/main/java/seremis/geninfusion/soul/gene/newAI/tradePlayer/GeneAITradePlayer.java
package seremis.geninfusion.soul.gene.newAI.tradePlayer;

import seremis.geninfusion.api.soul.IAllele;
import seremis.geninfusion.api.soul.lib.Genes;
import seremis.geninfusion.soul.Gene;
import seremis.geninfusion.soul.MasterGene;
import seremis.geninfusion.soul.allele.AlleleBoolean;

public class GeneAITradePlayer extends MasterGene {

    public GeneAITradePlayer() {
        addControlledGene(Genes.GENE_AI_TRADE_PLAYER_INDEX);
    }

    @Override
    public Class<? extends IAllele> possibleAlleles() {
        return AlleleBoolean.class;
    }
}

File: src/main/java/seremis/geninfusion/soul/gene/newAI/GeneAITradePlayer.scala
package seremis.geninfusion.soul.gene.newAI

import seremis.geninfusion.api.soul.lib.Genes
import seremis.geninfusion.soul.MasterGene

class GeneAITradePlayer extends MasterGene(classOf[Boolean]) {
    addControlledGene(Genes.GENE_AI_TRADE_PLAYER_INDEX)
}
